cli/sawtooth_cli/submit.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import logging
+import time
+
+from sawtooth_cli.rest_client import RestClient
+import sawtooth_cli.protobuf.batch_pb2 as batch_pb2
+
+LOGGER = logging.getLogger(__file__)
+
+
+def _split_batch_list(batch_list):
+    new_list = []
+    for batch in batch_list.batches:
+        new_list.append(batch)
+        if len(new_list) == 100:
```

Comment:
This should either be a named constant, or add a CLI flag `--batch-size-limit` that defaults to 100.